scale/docs/architecture/index.rst

```diff
@@ -11,6 +11,7 @@ Architecture
    overview
    jobs/index
    logs
+   messaging
```

Comment:
I believe this is included as it is chained from the root index.rst -> architecture.rst -> messaging.rst